sources/reference/tutorials/batch.md

```diff
@@ -0,0 +1,606 @@
+* Select `Create compute environment`
+
+
+### 6. Create your Batch Job queue
+
+When you submit Jobs, Batch uses a Job queue to manage what is and needs to be
+running and where it needs to run.
+
+* Navigate to the Batch console and select `Job queues`
+* Select `Create`
+* Give a name for the Job queue
+    * For the purpose of this guide we'll use `ctk-batch-job-queue`
+* Leave priority as `1`
+* Add a tag - We recommend at minimum, adding a tag `{"OWNER": "your-name"}`
+* In the `Connected compute environments` section, select your Compute environment
+from the dropdown
+* Select `Create`
+
+### 7. Create your Batch Job execution role
+
+Because you've set up an ECR repository with your Docker image in, you need to
+provide Batch with an execution role that will allow it to pull the image from
+ECR. It will also enable Batch to output the logs of the container to CloudWatch.
+
+Don't be confused when we refer to `Elastic Container Service`, Batch is using
+it under the hood.
+
+* Navigate to the IAM console and select `Roles`
+* Select `Create role`
+* Select `Elastic Container Service` from the list
+* Select `Elastic Container Service Task` from the `Select your use case` list
+* Select `Next: Permissions`
+* In the search bar, type `AmazonECSTaskExecutionRolePolicy` and select it
+* Select `Next: Tags`
+* Add a tag - We recommend at minimum, adding a tag `{"OWNER": "your-name"}`
+* Select `Next: Review`
+* Provide a name for the Role
+    * For the purpose of this guide we'll use `ctk-batch-execution-role`
+* Select `Create Role`
+
+### 8. Create your Batch Job job role
+
+The nature of `chaostoolkit-aws` means that we use `boto3` to make AWS requests
+within our experiment. To be able to make these calls, the container that is
+running our experiment needs credentials and permissions to do so.
+
+By creating a job role for our Job, we can:
+
+* Provide our Job with credentials with AWS
+* Outline exactly _what_ our Job is allowed to do
+
+* Navigate to the IAM console and select `Roles`
+* Select `Create role`
+* Select `Elastic Container Service` from the list
+* Select `Elastic Container Service Task` from the `Select your use case` list
+* Select `Next: Permissions`
+* Select `Create policy`
+* Move to the `JSON` tab and paste the following:
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeInstance*"
+            ],
+            "Resource": "*"
+        }
+    ]
+}
+```
+* Select `Next: Tags`
+* Add a tag - We recommend at minimum, adding a tag `{"OWNER": "your-name"}`
+* Select `Next: Review`
+* Provide a name for the Policy
+    * For the purpose of this guide we'll use `ctk-batch-job-policy`
+* Select `Create Policy`
+* Navigate back to the Role tab
+* In the search bar, type `ctk-batch-job-policy` and select it (You may have to
+click the refresh button)
```

Comment:
Should we be searching for a policy amongst the roles?

Reply:
The `Create Policy` button should have made a new window *from* the add policy page, then when you have made that policy and navigate back to the Role tab, you'll still be on that?

Reply:
Right, got it
I didn't realise it opened a new tab to sort out the policy - might it be worth explicitly mentioning it?

Reply:
https://github.com/chaostoolkit/chaostoolkit-documentation/pull/139/commits/8463cee19c2710875ea5deccd295ea25ff68c6ed

Done